jmix-rest/sample-rest/src/test/resources/application.properties

```diff
@@ -32,4 +66,6 @@ jmix.data.dbms-type-db1=hsql
 
 jmix.core.available-locales=en,ru
 
-main.liquibase.change-log=io/jmix/samples/rest/liquibase/test-changelog.xml
\ No newline at end of file
+#todo which one is correct? revert if removing unnecessary hsql file db will help
```

Comment:
What does it mean?

Reply:
It looks like v1.6 did not use this changelog. At least `io/jmix/samples/rest/liquibase/test-changelog.xml` has been used instead which has no include for `io/jmix/samples/rest/changelog.xml`. 

Replaced unclear message with more clear one, reference to jmix-framework/jmix#3758 has been added to deal with it during work on this issue.